readinglist/tests/test_views_article.py

```diff
@@ -82,6 +82,21 @@ def test_string_filters_naively_by_value(self):
         resp = self.app.get('/articles?title=MoFo', headers=self.headers)
         self.assertEqual(len(resp.json['items']), 6)
 
+    def test_different_value(self):
+        resp = self.app.get('/articles?not_status=2', headers=self.headers)
+        values = [item['status'] for item in resp.json['items']]
+        self.assertTrue(all([value != 2 for value in values]))
+
+    def test_minimal_value(self):
+        resp = self.app.get('/articles?min_status=2', headers=self.headers)
+        values = [item['status'] for item in resp.json['items']]
+        self.assertTrue(all([value >= 2 for value in values]))
+
+    def test_maximal_value(self):
+        resp = self.app.get('/articles?max_status=2', headers=self.headers)
+        values = [item['status'] for item in resp.json['items']]
+        self.assertTrue(all([value <= 2 for value in values]))
+
 
```

Comment:
``/articles?madmax_status=2`` will fail with PR #44 since this field will be unknown.

If the regexp is too permissive, prove it with a test :)